.travis.yml

```diff
@@ -12,7 +12,9 @@ install:
   - pip3 install coveralls
 
 before_script:
+  - python manage.py collectstatic
```

Comment:
Also, add a `python manage.py migrate` for the migrations.

Reply:
@gitaumoses4 kindly add the command above.

Reply:
Hello @esirK I have added the command, though I had to include the `psql -c 'create database'` command in order for it to function.